lib/go/test/test.go

```diff
@@ -251,6 +251,56 @@ func assertEqual(t *testing.T, expected, actual interface{}) bool {
 	return assert.Fail(t, message)
 }
 
+func assertCadenceArrayEqual(t *testing.T, expected, actual []cadence.Value) bool {
```

Comment:
We can't check order because we have no guarantee about ordering of elements in the arrays and dictionaries that we're having to verify. I'm only using these to verify arrays from dictionary keys in the staking contract, which never have duplicates, so maybe I should just put these methods in `staking_test_helpers` since they aren't general